sql/migrations/20200507205634_world.sql

```diff
@@ -0,0 +1,30 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20200507205634');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20200507205634');
+-- Add your query below.
+
+
+-- require alchemy skill of 50 to unlock expert alchemist training gossip
+-- tally berryfizz
+UPDATE `gossip_menu_option` SET `condition_id` = 361 WHERE `menu_id` = 4123 AND `option_id` = 5;
+
+-- yelmak
+UPDATE `gossip_menu_option` SET `condition_id` = 361 WHERE `menu_id` = 4126 AND `option_id` = 5;
+
+
+-- require alchemy skill of 225 to unlock master alchemist training gossip
+-- kylanna windwhisper
+UPDATE `gossip_menu_option` SET `condition_id` = 362 WHERE `menu_id` = 4133 AND `option_id` = 5;

```

Comment:
Condition 362 is for 125 skill. Did you mean to use 363 which is for 200 skill? This npc is a Master Alchemist.

Reply:
@ratkosrb I believe you need to have 225 skill to unlock master alchemist gossip, I may need to add a condition?